R/merge_kallisto.R

```diff
@@ -139,7 +139,8 @@ if (!is.null(anno)) {
   # Write log
   if ( verb ) cat("Generating table of transcript to gene IDs....\n", sep="")
   # Generating two-column data.frame linking transcript id (column 1) to gene id (column 2).
-  tx2gene <- tr2g_gtf(anno, gene_name=NULL, transcript_version=NULL)
+  df <- readGFF("../input_files/homo_sapiens/annotation.gtf", tags=c("gene_id","transcript_id"))
```

Comment:
@paulaidt This part is hardcoded ("../input_files/homo_sapiens/annotation.gtf"). Can you please update it and provide the `anno` as input in the function?